src/data/option/basic.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2017 Mario Carneiro. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro
 -/
-import logic.basic data.bool data.option.defs tactic.interactive
+import logic.basic data.bool data.option.defs tactic.ext tactic.simpa
```

Comment:
I don't like this change. Did you take `ext` and `simpa` out of `tactic.interactive`? Files should import `tactic.interactive` unless they need to work around a particular dependency problem (or they are before `tactic.interactive` in dependency order).

Reply:
I don't think that's a good standard to have. You're enforcing excessive coupling. When you're writing the file, that's fine to just import `tactic` but then `olean-rs` will tell you how to reduce the coupling. With the old `tactic.interactive`, you're making sure that every time `data.list` changes, everything has to be recompiled even if it doesn't need `data.list`